drv/stm32xx-sys-api/src/h7.rs

```diff
@@ -57,9 +57,7 @@ pub enum Peripheral {
     OctoSpi2 = periph(Group::Ahb3, 19), // B3 only
     Sdmmc1 = periph(Group::Ahb3, 16),
 
-    #[cfg(feature = "h7b3")]
-    OctoSpi1 = periph(Group::Ahb3, 14), // B3 only
-    #[cfg(any(feature = "h743", feature = "h747", feature = "h753"))]
+    #[cfg(any(feature = "h743", feature = "h753"))]
```

Comment:
Is `any(feature = "h743", feature = "h753")` guaranteed to be true?  Now that we've removed `h7b3` support, `h743` and `h753` are the only remaining `h7*` features, so I think we can remove this condition (and others below).